modules/data.atmosphere/inst/scripts/extract2driver.R

```diff
@@ -189,12 +189,12 @@ potentialRadiation <- function(day,time,LAT,timelag){
 			0.000719 * cos(2.0*dayangle)+
 			0.000077 * sin(2.0*dayangle)
 	solartime=time/secsInHour-12.0+timelag
-	radiation = 1367.0 * 
-			eccentricity * 
-			(cos(declination) * 
-				cos(deg2rad(LAT)) * 
+	radiation = 1367.0 *
```

Comment:
magic number !!!also uses `=` instead of `<-`

Reply:
This is an old script (not a package function definition) that hasn't been touched in a while, so I'm just gonna leave it. There's a lot more that's wrong with it, which I don't feel like spending time fixing.